openlibrary/plugins/openlibrary/js/validate.js

```diff
@@ -28,9 +30,7 @@ export default function initValidate() {
     );
 
     $.validator.messages.required = '';
-    // _ is defined in openlibrary\plugins\openlibrary\js\i18n.js
-    // eslint-disable-next-line no-undef
-    $.validator.messages.email = _('Are you sure that\'s an email address?');
+    $.validator.messages.email = ugettext('Are you sure that\'s an email address?');
```

Comment:
Can these be removed now in index.js?
![image](https://user-images.githubusercontent.com/6251786/64910847-fb657d80-d6e8-11e9-8194-74cb7518fc7d.png)

Reply:
I'm a bit scared to as it's really difficult to grep "_(" given python and I'm pretty sure these are still in usage.

Reply:
See also #2353 - we need to work out what i18n if any is happening in the codebase.

Reply:
:+1: